src/components/MenuBubble.vue

```diff
@@ -159,15 +174,14 @@ export default {
 			command({ href: null })
 		},
 		bubblePosition(menu) {
-			// below the first line, above all others
-			const vertical = menu.top < 45
-				? { top: `${menu.top}px` }
-				: { bottom: `${menu.bottom}px` }
+			const wrapper = this.$parent.$refs.wrapper
```

Comment:
Let's keep in mind this breaks if the component structure changes.

Reply:
I could not pass scrollTop in because it changes whenever the text is scrolled up and down - but it does not seem to be reactive.
I'm now passing in the content wrapper explicitely. That should allow using the Menu Bubble in other contexts, and is less magical than requiring the parent to define a `ref` called `wrapper`

Reply:
:+1: